### DISCUSSION TITLE
@vuepress/plugin-sitemap 安装之后无法启动问题

### DISCUSSION THREAD

**Discussion by kk305cs** [vuepress/ecosystem#76] [Q&A] [answered]
请问@vuepress/plugin-sitemap v2.0.0 rc3 是和 vuepress2 哪个版本配合的呢？我安装之后 运行代码会报一个错
<img width="2170" alt="image" src="https://github.com/vuepress/ecosystem/assets/26682400/0dc0fcea-18ee-4689-9b2f-406e93053964">

**Comment by Mister-Hope** [comment, marked as answer]
最新版都是相互搭配的，都升到最新发布的版本就好了

**Reply by kk305cs** [reply]
谢谢 已全部更新 就能安装成功了
